README.md

```diff
@@ -1,6 +1,6 @@
 # IMPORTANT: Latest Version
 
-The current version is 4.0.3. Please see the [changelog](./CHANGELOG.md) for details on version history.
+The current version is 4.0.4. Please see the [changelog](./CHANGELOG.md) for details on version history.
```

Comment:
Can we also update the section titled "Add the Authentication Plugin to the Application" below?

Reply:
Ok will do